frodo.js

```diff
@@ -1,72 +1,67 @@
+const formattedDate = new Date().toLocaleDateString();
```

Comment:
SCRIPT-This logic might be clearer if you used a switch statement instead of multiple if-else statements.